.github/workflows/ci.yml

```diff
@@ -17,7 +17,7 @@ jobs:
 
       - uses: actions/setup-node@v2
         with:
-          node-version: '12.14.0'
+          node-version: '16.13.0'
```

Comment:
I wrote up some thoughts on library testing stuff in https://github.com/Shopify/loom/pull/305#pullrequestreview-794776059

In order not not cause breaking changes I'd love to see us say "we still support v12/v14/v16" for now, and run CI against multiple node versions, like what Quilt does -  https://github.com/Shopify/quilt/pull/2063/files#diff-be891c5470f83d305ee76f877cb1db28b80951b85f42dae02e9fb8ad44df6886R14